### DISCUSSION TITLE
This dialogue is abandoned, please see https://github.com/bigtreetech/CB1/discussions/9

### DISCUSSION THREAD

**Discussion by bigtreetech** [bigtreetech/CB1#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Will-wastelander** [comment]
What do you want to discuss?  Everyone is asking for this. It should he a given that an image created with an LTS Devian 11 release is created and released by @bigtreetech as the creator of the hardware.

**Reply by bigtreetech** [reply]
Hello. please see https://github.com/bigtreetech/CB1/discussions/9

**Comment by WhiskeySheppard** [comment]
Finally, a working and updated OS for the CB1.

**Comment by hyptx** [comment]
I am having some troubles configuring the TFT35 SPI screen from BTT. There is virtually no instructions. I was sent the manual but all it has is a pinout. How to I turn this on. Manta M8P. I just get a white screen and random black or random pixels. I2C write errors. I have uncommented the overlays= in the BoardENV file.

**Reply by Pneumanifest** [reply]
Which overlays did you uncomment? If you are only setting it up for the screen then you just need the 
    overlays=tft35_spi
 in the BoardEnv.txt file

**Reply by Pneumanifest** [reply]
Also which CB1 do you have? The V2.1 needs to use the IO2CAN board according to the listing.

**Reply by hyptx** [reply]
Yes, I uncommented tft35_spi. I know what the listing says, but i'm plugging into the SPI display header on the Manta M8P with the cb1 mounted to it, so I'm not using one of those can boards. I think I just need to configure the pins somewhere. Not sure.

**Reply by Pneumanifest** [reply]
Shouldn't need to reconfigure anything it boots right up. All I did was uncomment the overly and nothing else.

**Reply by hyptx** [reply]
It is the 2.1 CB1, if for some reason that doesn't work I'm going to be upset, it's brand new stuff.

**Reply by Pneumanifest** [reply]
The v2.1 needs the IO2CAN board

**Reply by hyptx** [reply]
Thanks much. I guess the lesson learned is check to make sure you are getting the v2.2 when ordering one of these.

**Comment by bendermh** [comment]
Hello,
I'm using a CB1 board in a Biqu-BX printer. 
It works nice, except for the display, I'm using the original 7'' display of the printer. The CB1 in console-text mode works nice and there is not problem to display the text console in the printer's display, but when I start to run a graphical enviorment in the display it starts to see very blurred (unable to be used).
 
How can I configure the CB1 to support the graphic display of the Biqu-Bx. I attach the Biqu specs to configure the Biqu-Bx display in raspbian. How can I set all these specs in the CB1 configurartion file ? 

https://github.com/bigtreetech/OctoBTT/raw/master/IMG/Ready/ConfigDisplayMode.jpg

**Comment by Ideefixes** [comment]
Hello,
I Have already issue to connect Manta M4P/CB1 2.2 through WIFI. Latest Kernel installed. I can reach the board trough Lan. Wifi seems to fonction but when i reboot CB1 Wifi seems to not be active. I say that because i can activate wifi connection under SSH with nmtui command and than BTT-CB1 appear with Wifi IP and i am abble to reach mainsail......but when i unplugg power or reboot the device in mainsail the wifi is unactivated. SSID and PW are rigth in system config file from SD card. Any Idee???? 
I tray so many things with no more results that i am thinking to forgive with CB1 and go for CM4 module.....

**Reply by Ideefixes** [reply]
Hello,
Finally i got it work....What I do : i install only the Kernel first, Through wire Lan i activated wifi with nmtui command. I also put in system file WIFI SSID and WIFI PW. Than CB1 was reacheable though WIFI. I install than Klipper/moonraker/mainsail/klipperscreen and crowsnet with KIAUH. I have some more issue with klippercreen at start but with github i had succesfull debugging.

**Comment by fugalster** [comment]
I'd like to try using the CB1 as a general purpose Raspberry Pi with a desktop environment. Will there be an image with e.g. PIXEL included?

**Comment by emersonsc** [comment]
Why is there no desktop version of the OS for the CB1 especially considering you make boards in the pi footprint now?

Also, why no OS update in almost a year? is the CB1 abandoned already...?

**Reply by frutabruta** [reply]
According to my information, BTT developers concentrate more on mainlining all the patches necessary to run any Linux on CB1, especially Armbian. Armbian is available now, has GUI and all the contemporary updates. But it is missing all the BTT specific scripts, like setting BTT PAD, SPI, ADXL345 etc.
[Armbian download page](https://www.armbian.com/bigtreetech-cb1/)

**Comment by mat93ut99** [comment]
I am trying to rotate the BTT HDMI 5 touch matrix which is connected to my CB1+MANTA 8P board.   I am running BTT CB1 V2.3.4 (CB1_Debian11_Klipper_kernel5.16_20240319.img) latest OS image.   I was able to rotate the screen by editing the system.cfg file, and I successfully rotated the screen 180-degrees by modifying the ks angle=inverted.   However, I have no idea on how to edit the touch screen matrix.  Following the Klipperscreen documentation, I run the `DISPLAY=:0 xinput` , but I think I should see a connected device.    

![Screenshot 2024-07-21 175330](https://github.com/user-attachments/assets/27997e8b-a6c9-4b0e-ad6f-ceb34a281430)
Can anyone point me in the right direction?

**Reply by frutabruta** [reply]
Undo any changes you have made in the software. The HDMI5 display can be rotated by pressing one of the hardware buttons on the side. It should also rotate the touch accordingly.

**Comment by Ireq007** [comment]
Hi, I'm new to the world of Vorona. I have a BTT Manta M8P+CB1 motherboard. I'm trying to flash CB1_Debian 12 version 3.0.0 to a microSD card. Every time I flash it and check it with the Rosbperry Imager, I get an error and it tries to re-flash the file. Any advice? I have Windows 11. I've also tried iOS and a MacBook Air, but I get the same error.
